src/pages/Reader.tsx

```diff
@@ -51,6 +51,15 @@ class Reader extends React.Component<any, State> {
     this.myRef = React.createRef();
   }
 
+  mangaName = (): string => {

```

Comment:
this is why we might want to store a metadata file along with the actual manga.

what if the name has punctuation and special characters and stuff:
```
darwins_game.zip -> Darwins Game
```

Reply:
Do you think I should create metadata now because we're gonna need it later -  maybe ill look into how other ppl do it

Reply:
o lol just saw the issue